opensrp-anc/src/main/assets/json.form/anc_register.json

```diff
@@ -56,9 +56,9 @@
     "fields": [
       {
         "key": "wom_image",
-        "openmrs_entity_parent": "",
+        "openmrs_entity_parent": "The image of the woman, if taken, should sync to her OpenMRS patient medical record.",
         "openmrs_entity": "",
-        "openmrs_entity_id": "",
+        "openmrs_entity_id": "--",
```

Comment:
The openmrs entity id with `--` should be changed to the correct entity ID or entity